packages/lab/src/pill-next/PillNext.css

```diff
@@ -16,7 +10,7 @@
   --pillNext-text-color-disabled: var(--salt-actionable-primary-foreground-disabled);
 }
 
-.saltPillNext-action {
+.saltPillNext {
   appearance: none;
   -webkit-appearance: none;
```

Comment:
Although this is copied from Button, it's actually not needed. `appearance` is supported by all of our supported browsers